docs/standard/components.md

```diff
@@ -1,80 +1,88 @@
 ---
-title: .NET Architectural Components
-description: Learn about key .NET architectural components, such as the .NET Standard, .NET runtimes, and tooling.
-keywords: .NET, .NET Standard, .NET Standard, .NET Core, .NET Framework, Xamarin, MSBuild, C#, F#, VB, compilers
+title: .NET architectural components
+description: Describes .NET architectural components such as the .NET Standard, .NET implementations, .NET runtimes, and tooling.
+keywords: .NET, .NET Standard, .NET Core, .NET Framework, Xamarin, MSBuild, C#, F#, VB, compilers
 author: cartermp
 ms.author: mairaw
-ms.date: 11/16/2016
+ms.date: 08/10/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: dotnet-standard
 ms.devlang: dotnet
 ms.assetid: 2e38e9d9-8284-46ee-a15f-199adc4f26f4
 ---
 
-# .NET Architectural Components
+# .NET architectural components
 
-.NET is made up of a number of key components.  It has a standard library, called the .NET Standard, which is a large set of APIs which runs everywhere.  This standard library is implemented by three .NET runtimes - .NET Framework, .NET Core, and Mono for Xamarin.  The .NET languages also run on any .NET runtime.  Additionally, there are tools on every platform which allow you to build projects.  These tools are the same regardless of your choice of runtime.
-
-Here's a graphical overview of each of the previously mentioned components of .NET and how they fit.
-
-![All .NET Architectural Components Together](media/components.png)
-
-What follows is a brief explanation of each of the key components shown above.  
+A .NET app is developed for and runs in one or more *implementations of .NET*.  Implementations of .NET include the .NET Framework, .NET Core, and Mono for Xamarin. There is an API specification common to all implementations of .NET that's called the .NET Standard. This article gives a brief introduction to each of these concepts.
```

Comment:
Not sure what "Mono for Xamarin" means here in the context of a list of implementations, see later comments.